### DISCUSSION TITLE
REM AC Power -- Switched vs. "Always On"

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#727] [REM] [answered]
I fear you will be getting weary of all the "discussions" I've started recently.   Sorry, but another...

My Intellichem was AC powered from the same relay as the pool pump  That's the way the manual says to wire it.

Thinking about, when the pump is off, there is no useful information that can be obtained--there's no water flow.  Really, wiring it this way makes the flow switch redundant.  

I think the only down side is the boot time of the rpi.  But, that's short relative to the minimum "mixing time"--the time the pump must be running before ORP and pH readings are considered valid.

Is this the way you guys wire it?

**Comment by rstrouse** [comment, marked as answer]
No I would not wire it that way.  REM uses software latches that keep the pump from running should any of the components fail or the flow switch is not engaged.  If power is lost the pump relay is connected to the normally open side of the pump.

**Comment by davthomaspilot** [comment]
This is just REM--power to dosing pumps only.  There's no control of power to the main pool pumps.

The rest of node-js pool controller is inside the house and is powered separately.  It communicates with Pentair EasyTouch via the RS485.

What component failures are you considering?  All the REM hardware can do is turn on a dosing pump. There's nothing to protect if the power to the main pump isn't on.

What am I missing?

**Comment by davthomaspilot** [comment]
It's coming back to me--

Back when I did some packet analysis on the data sent by Intellichem, I now vaguely recall that njsPc got bogus Intellichem data from EasyTouch, and didn't have a way to know the Intellichem unit was unpowered.

So, I'll hook it up always powered on.

But, I think you can pretty much eliminate the need for a flow switch if you wire it so no power gets to the dosing pumps unless the pool/spa pumps are on.  Not sure why Intellichem has the flow switch AND has you wire it in parallel with the pool pump.  Redundancy, I guess.

**Comment by rstrouse** [comment]
Not really the flow switch is really about pump failure or cavitation.
